src/test/java/features/89 API Testing.feature

```diff
@@ -0,0 +1,65 @@
+#language: en
+@89
+Feature: 89 API Testing
+  Tests using the Swagger Pet Store example API.
+  https://petstore3.swagger.io/
+      
+  @89A
+  Scenario: 89A POST Swagger Test
+    Given I use the API named Pet Store API
+    When I load puppydata to use as the API body

```

Comment:
same thing here. To me, the phrase "API body" doesn't really make sense

Reply:
Fixed, as well as in the load request body step.